### ISSUE TITLE
invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by savushkin-yauheni** [lc/gau#114]
Hi!

After new release I constantly got an error: invalid memory address or nil pointer dereference

```
./gau  --pages 1 --retries 1 --o /tmp/adsasddas https://www.google.com

WARN[0000] error reading config: open /Users/savik/.gau.toml: no such file or directory 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x18 pc=0x100a758c0]

goroutine 5 [running]:
github.com/lc/gau/v2/pkg/output.WriteURLs({0x0, 0x0}, 0x0?, {0x100dd7b18, 0x140002ce930}, 0x0)
        /Users/savik/own/projects/temp/gau/pkg/output/output.go:34 +0x310
main.main.func1(0x0?)
        /Users/savik/own/projects/temp/gau/cmd/gau/main.go:52 +0x90
created by main.main in goroutine 1
        /Users/savik/own/projects/temp/gau/cmd/gau/main.go:48 +0x3b8

```

Is it only for me?

Thanks

**Comment by lc** [comment]
Hey @savushkin-yauheni,

Not only you – investigating why this is occurring!

**Comment by lc** [comment]
Fixed! `v2.2.1` should be done building shortly
